src/java/org/apache/cassandra/cql3/Lists.java

```diff
@@ -52,6 +59,8 @@
  */
 public abstract class Lists
 {
+    private static final Logger logger = LoggerFactory.getLogger(Lists.class);
```

Comment:
I can remove the unused loggers but it's pretty handy to have the logger already declared rather than having to go and manually bring it back. I think it's worth the one line, but don't feel strongly about it.

Reply:
Added @SuppressWarnings("unused")